### DISCUSSION TITLE
Error launching codespace

### DISCUSSION THREAD

**Discussion by ghost** [community/community#29514] [Codespaces] [answered]
Hello guys, I get an annoying error while launching codespace: 
VSCode Workbench mount failed with "" , nothing else, no script description, nothing. I'm sure some of you already experiance that bug, restarting a codespace seems to do nothing.
May I ask for some help with solving that issue?

![Zrzut ekranu (2160)](https://user-images.githubusercontent.com/57498794/184622213-900d313a-b6ed-4f16-b764-b891011bc0cc.png)

**Comment by thomas-sickert** [comment, marked as answer]
Sorry you had an issue! This was related to a [recent outage](https://www.githubstatus.com/incidents/0hkbgpcpbkc6) and should be resolved now. Please let us know if you are still experiencing problems.

**Reply by ghost** [reply]
Hi Thomas, I would like to thank you for helping me with my case. Everything works fine at the moment.
Thank You so much...You're a legend!

**Reply by avhwb** [reply]
Hi! I am still experiencing this problem. Restarting my codespace does not help. Please help me!

**Comment by ayointegral** [comment]
Mine is still not working with the `Restarting codespace error` showing repeatedly. Please help.
